src/executables/csql_result_format.c

```diff
@@ -1479,6 +1535,19 @@ csql_db_value_as_string (DB_VALUE * value, int *length, bool plain_string)
       db_private_free (NULL, json_body);
       if (result)
 	{
+	  if (output_type == CSQL_QUERY_OUTPUT || output_type == CSQL_LOADDB_OUTPUT)
+	    {
+	      char *new_result;
+
+	      new_result =
+		string_to_string (result, column_enclosure, '\0', strlen (result), &len, false, change_single_quote);
+
+	      if (new_result)
+		{
+		  free (result);
+		  result = new_result;
+		}
+	    }
```

Comment:
Because this style of same code appears several times, approximately 9 times, how about replacing code with a macro or a function?

Reply:
thank you to comment, but i would be better to keep this code.